proxima-chapter-1.md

```diff
@@ -228,92 +230,107 @@ Then, let's proceed to prepare the environment for subsequent steps like:
+     
+    In a Matrix Terminal session, subscribe to all topics with `mosquitto_sub -h 127.0.0.1 -p -t '#' -u <YourUsername> - <YourPassword>`.
+5. In another Matrix Terminal session, publish a message with:
+`mosquitto_pub -h 127.0.0.1 -p <port>-t 'Test' -u <YourUsername> -P <YourPassword> -m '{"Name":"Jack Torrance"}'`.
+6. If all is well, you should see the message on the subscriber's terminal session: in such a case, grab a coffee or a nice cup of tea, as we're going secure our MQTT Server first, and then installling the OCI SDK and examples for Python.
 
 #### Secure the MQTT Server running on OCI Compute
 
 There's a number of guides and tutorials, on the Web, about configuring and using the MQTT TLS security, either via [Let's Encrypt](https://letsencrypt.org/?source=:ex:of:::::RC_WWMK210721P00049:Medium) or just creating your own CA, Server Keys and certificates. What I did here is follow the shortest path (creating my own stuff), but feel free to follow your creativity and developer flair. 
 
 **Note: when entering the country, organization and other info donʼt use the same exact information for the CA and the Server Certificate as you might encounter problems.**
 
-- Generate a CA key
+1. Generate a CA key
     1. openssl genrsa -des3 -out ca.key 4096 and choose a password, in my case password1,
     2. openssl req -new -x509 -days 2112 -key ca.key -out ca.crt
-- Generate a server key
-    1. openssl genrsa -des3 -out server.key 4096 and choose another password, in my case password2
-- Generate a Certificate Signing Request
-    1. openssl req -new -key server.key -out server.csr
-- Sign the Certificate Signing Request
-    1. openssl x509 -req -days 365 -in server.csr -CA ca.crt -CAkey ca.key -set_serial 01 -out server.crt
-- Verify that every bit is ok
-    1. openssl rsa -noout -text -in server.key
-    2. openssl req -noout -text -in server.csr
-    3. openssl rsa -noout -text -in ca.key
-    4. openssl rsa -noout -text -in ca.key
-
-- Make a version of the server key that doesn't need the password
-    1. openssl rsa -in server.key -out server.key.insecure
-    2. mv server.key server.key.secure
-    3. mv server.key.insecure server.key
-
-- Edit the file /usr/lib/python3.6/site-packages/certifi/cacert.pem adding the ca.crt file content (PEM public Key)
+2. Generate a server key
+    - openssl genrsa -des3 -out server.key 4096 and choose another password, in my case password2
+3. Generate a Certificate Signing Request
+    - openssl req -new -key server.key -out server.csr
+4. Sign the Certificate Signing Request
+    - openssl x509 -req -days 365 -in server.csr -CA ca.crt -CAkey ca.key -set_serial 01 -out server.crt
+5. Verify that every bit is ok
+    
+    ```sh
+    $ openssl rsa -noout -text -in server.key
+    $ openssl req -noout -text -in server.csr
+    $ openssl rsa -noout -text -in ca.key
+    $ openssl rsa -noout -text -in ca.key
+    ```
```

Comment:
You can use the indented paragraph trick from above with fenced code blocks as well.